test/unit/murmurhash3_spec.js

```diff
@@ -0,0 +1,53 @@
+/* globals jasmine, expect, it, describe, MurmurHash3_64 */
+'use strict';
+
+describe('MurmurHash3_64', function() {
+
+  it('instantiates without seed ', function() {
+      var hash = new MurmurHash3_64();
+      expect(hash).toEqual(jasmine.any(MurmurHash3_64));
+  });
+  it('instantiates with seed ', function() {
+      var hash = new MurmurHash3_64(1);
+      expect(hash).toEqual(jasmine.any(MurmurHash3_64));
+  });
+
+  var hexDigestExpected = 'f61cfdbfdae0f65e';
+  var sourceText = 'test';
+  var sourceCharCodes = [116,101,115,116]; // 't','e','s','t'
+  it('generates expected from string', function() {
```

Comment:
I would change this to `correctly generates a hash from a string` (and `Uint8Array`/`Uint32Array` below) as it's a bit more descriptive in the logs when running the unit tests.